src/gatsby-theme-carbon/components/LeftNav/ResourceLinks.js

```diff
@@ -8,7 +8,7 @@ const links = [
     },
     {
         title: 'Design Kit',
-        href: '/resources#theme-libraries',
+        href: '/resources#IBM.com Library',
```

Comment:
```suggestion
        href: '/resources#ibm.com-library',
```

@JennySanchez I tried the link, but I think we did need the dash in the link.